pkgs/top-level/darwin-packages.nix

```diff
@@ -118,6 +118,8 @@ makeScopeWithSplicing' {
 
         openwith = callPackage ../os-specific/darwin/openwith { };
 
+        sleepwatcher = callPackage ../os-specific/darwin/sleepwatcher { };
```

Comment:
Despite being Darwin-specific this should go in `pkgs/by-name`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md